web-dev-server.config.js

```diff
@@ -51,4 +51,8 @@ module.exports = {
     },
     esbuildPlugin({ ts: true }),
   ],
+  nodeResolve: {
+    // Use Lit in production mode
+    exportConditions: ['default'],
```

Comment:
This is mainly to prevent spamming Lit dev mode warnings. We can change it later.